db/db_write_test.cc

```diff
@@ -185,66 +185,6 @@ TEST_P(DBWriteTest, LockWalInEffect) {
   ASSERT_OK(dbfull()->UnlockWAL());
 }
 
-TEST_P(DBWriteTest, MultiThreadWrite) {
```

Comment:
Can we keep the test? If the test is broken with multi-batch write,  simply disable it for multi-batch write.

Reply:
I did not remove it. I just moved it into `db/db_multi_batch_write_test.cc`.